inscriptions/ISic001266.xml

```diff
@@ -190,32 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140755">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    
-                    <expan>
-                        <abbr>Θ</abbr>
-                        <ex>εοῖς</ex>
-                    </expan>
-                    <expan>
-                        <abbr>Κ</abbr>
-                        <ex>α</ex>
-                        <abbr>τ</abbr>
-                    </expan>
-                    (αχθονίοις).<lb n="2"/>Χαίρο<supplied reason="omitted">υ</supplied>σα αἰ-<lb n="3"/>εὶ γλυκυτάτη<lb n="4"/>ζήσασα ἔτη
-                    <supplied reason="lost">—</supplied>
-                    <lb n="5"/>πατὴρ τ
-                    <supplied reason="lost">ῇ ἰδίᾳ</supplied>
-                    <lb n="6"/>
-                    <supplied reason="lost">θυ</supplied>γατρ<supplied reason="lost">ὶ ἐποίησεν</supplied>.
-                    <lb n="7"/>
+                    <lb n="1"/><w><expan><abbr>Θ</abbr><ex>εοῖς</ex></expan></w> <w><expan><abbr>Κ</abbr><ex>α</ex><abbr>τ</abbr><ex>αχθονίοις</ex></expan></w>
+                    <lb n="2"/><w>Χαίρο<choice><corr>υ</corr><sic>ι</sic></choice>σα</w> <w>αἰ
```

Comment:
I think one must make a choice here between either: 
a) `<w>Χαίρο<unclear>υ</unclear>σα</w>`
b)`<w>Χαίρουσα</w>` and just reporting Χαίροισα in the apparatus as recorded originally by Dorville.
Note that the same is actually true for the 't' in the first line as well (an i read by Dorville).
My inclination in this case is to report it with 'unclear' as here, and then in the apparatus to report Dorville's reading in Kaibel (since otherwise the text looks reliable, and both these cases are easily explained by damage to the stone meaning only a vertical stroke was visible).